eng/mgmt/mgmtmetadata/recoveryservices-backup.txt

```diff
@@ -0,0 +1,14 @@
+﻿Installing AutoRest version: latest
+AutoRest installed successfully.
+Commencing code generation
+Generating CSharp code
+Executing AutoRest command
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/recoveryservices-backup/readme.md --csharp --version=latest --reflect-api-versions --tag=package-2017-07 --csharp-sdks-folder=C:\Users\prupadhy\Desktop\a\azure-sdk-for-net\sdk
+2020-06-09 10:08:41 UTC
+Azure-rest-api-specs repository information
+GitHub fork: Azure
+Branch:      master
+Commit:      8e5820c94a1ea71f97cd98a1125b95d165cb26a1
+AutoRest information
+Requested version: latest
+Bootstrapper version:    autorest@3.0.6187
```

Comment:
After you update `generate.ps1` and generate again, here should be `autorest@2.0.xxxx`